### PR TITLE
Add an article about How can I give a group discount promotion

### PR DESCRIPTION
**Pull request by akshit15013**
Description : Add an article about How can I give a group discount promotion
Issue No. : #23

### CODE REVIEW

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@tabesin

File: event-setup/How-can-I-give-a-group-discount-promotion.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# How can I give a group discount promotion
+
+To give a group discount promotion, [create a event first](/event-setup/create-event.md).
```
Comment (Ryan10145) [inline event-setup/How-can-I-give-a-group-discount-promotion.md]:
```suggestion
To give a group discount promotion, [create an event first](/event-setup/create-event.md). 
```

File: event-setup/How-can-I-give-a-group-discount-promotion.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Now, open the Tickets section.(see the picture below)
+
+![Tickets Section](/event-setup/images/ticket-section.png)
+
+Then open the Discount Codes section and click on Create a Discount option.(see the picture below)
+
+![Discount Codes Section](/event-setup/images/discount-section.png)
+
+Enter a Discount Code and enter the discount amount in US$ or percentage form. Enter a value in the NUMBER OF DISCOUNT TICKETS.Click on save discount option.(see the picture below)
```
Comment (Ryan10145) [inline event-setup/How-can-I-give-a-group-discount-promotion.md]:
```suggestion
Enter a Discount Code and enter the discount amount in US$ or percentage form. Enter a value in the NUMBER OF DISCOUNT TICKETS. Click on save discount option.(see the picture below)
```

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@Masquerade0097 @abishekvashok

**Comment by akshit15013** [comment]
@mariobehling

**Comment by akshit15013** [comment]
Please merge my PR

**Comment by akshit15013** [comment]
@mariobehling @abishekvashok Please merge my PR. 
It has been a lot of time o had been waiting for it to get merged.

**Comment by rpotter12** [comment]
Is your fork updated??

**Pull request merged**